lib/spack/spack/cmd/create.py

```diff
@@ -41,16 +38,37 @@
 from spack.spec import Spec
 from spack.util.naming import *
 from spack.repository import Repo, RepoError
-import spack.util.crypto as crypto
 
 from spack.util.executable import which
-from spack.stage import Stage
 
 
 description = "Create a new package file from an archive URL"
 
-package_template = string.Template(
-    _copyright + """
+package_template = string.Template("""\
```

Comment:
Flake8 didn't like it when there was any code before the import statements. I can remove it from the top if you want.